src/server/lib/deeplink/oktaRecoveryToken.ts

```diff
@@ -0,0 +1,97 @@
+import { getApp } from '@/server/lib/okta/api/apps';
```

Comment:
I felt like just using `deeplink` didn't adequately describe the usage, while `deeplink/oktaRecoveryToken` did. Which was the main reasoning.

Reply:
This way you know it's something related to deeplinking, and also related to the recovery token, rather than one or the other, without having to go into the file.

Reply:
This seems reasonable.